nixos/doc/manual/release-notes/rl-2405.section.md

```diff
@@ -391,6 +391,11 @@ The pre-existing [services.ankisyncd](#opt-services.ankisyncd.enable) has been m
 
 - The `erlang-ls` package no longer ships the `els_dap` binary as of v0.51.0.
 
+- The `conda` package has been updated to 24.1.2 and is now built from source. This package no longer relies on entering a shell to install miniconda. Running `conda init` is still required.
+  By default, conda environments will be installed in `~/.conda/envs` and conda packages will be cached in `~/.conda/pkgs`
```

Comment:
Please use XDG directories or is that an upstream default?

Reply:
this is an upstream default.